engine/install/linux-postinstall.md

```diff
@@ -177,7 +177,7 @@ Configuring Docker to accept remote connections can be done with the `docker.ser
     ExecStart=/usr/bin/dockerd -H fd:// -H tcp://127.0.0.1:2375
     ```
 
-3. Save the file.
+3. Save the file. ( this will create `/etc/systemd/system/docker.service.d/override.conf` )
```

Comment:
I'm a bit on the fence if we should add this detail (a bit worried that it slightly distracts from the steps itself).


If we _do_ want to keep this, perhaps put it below the instruction itself (and avoid using future tense, which is generally avoided in technical documentation)


```suggestion
3. Save the file.

    The file is saved as `/etc/systemd/system/docker.service.d/override.conf`.
```

Reply:
I thought it would be useful to someone.
If this violates docs standard then we throw away this.